### PR TITLE
SpinKit

### PR DESCRIPTION
**Pull request by teohhanhui**

### CODE REVIEW

**Comment by jimaek** [comment]
Can these spinners be used as is by other people? It looks more like a demo files which doesnt make sense to host

**Comment by teohhanhui** [comment]
It's a collection of ready-to-use pure CSS spinners (animation, transform).

File: files/spinkit/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Tobias Ahlin"
+github = "https://github.com/tobiasahlin/SpinKit"
+homepage = "http://tobiasahlin.com/spinkit/"
+description = "A collection of loading indicators animated with CSS"
+mainfile = "css/spinkit.css"
```
Comment (jimaek) [inline files/spinkit/info.ini]: The correct here would be `spinkit.css`

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge please

**Pull request closed without merging**